src/foam/nanos/crunch/ui/CapabilityWizardlet.js

```diff
@@ -42,8 +42,8 @@ foam.CLASS({
         if ( ! this.of ) return null;
 
         var ret = this.of.getAxiomByName('capability') ?
-          this.of.create({ capability: this.capability }, this) :
-          this.of.create({}, this);
+          this.of.create({ capability: this.capability }, this.__subContext__) :
+          this.of.create({}, this.__subContext__);
```

Comment:
How is this different than just using 'this'?

Reply:
Is it not meant to be? After I saw the error I thought I did that by accident and this was supposed to be a context rather than an element. If elements are meant to work as the second arg then this may be a deeper issue.